src/main/java/net/ccbluex/liquidbounce/features/module/modules/player/Blink.kt

```diff
@@ -56,14 +63,37 @@ object Blink : Module("Blink", ModuleCategory.PLAYER, gameDetecting = false) {
         if (mc.thePlayer == null)
             return
 
-        when (packet) {
-            is C04PacketPlayerPosition, is C06PacketPlayerPosLook, is C08PacketPlayerBlockPlacement,
-            is C0APacketAnimation, is C0BPacketEntityAction, is C02PacketUseEntity -> {
-                event.cancelEvent()
-                packets += packet
+        when (mode.lowercase()) {
+            "sent" -> {
+			    if (event.eventType == EventState.RECEIVE && isCleaning == 0) {
+                    handlePackets(*packetsReceived.toTypedArray())
+                    packetsReceived.clear()
+                }
```

Comment:
What is wrong with formating of all of these?

Reply:
> What is wrong with formating of all of these?

Everything